google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -74,37 +69,51 @@ class _SecurityCenterOrganizationsFindingsRepository(
 
     def __init__(self, **kwargs):
         """Constructor.
+
+        Args:
+            **kwargs (dict): The args to pass into GCPRepository.__init__()
         """
+
+        LOGGER.debug(
+            'Creating _SecurityCenterOrganizationsFindingsRepositoryClient')
         super(_SecurityCenterOrganizationsFindingsRepository, self).__init__(
-#            component='securitycenter.organizations.findings', **kwargs)
             component='organizations.findings', **kwargs)
 
 
 class SecurityCenterClient(object):
     """Cloud Security Command Center Client.
+
     https://cloud.google.com/security-command-center/docs/reference/rest
     """
 
-    def __init__(self, **kwargs):
-        """Initialize.
-        Args:
-            global_configs (dict): Forseti config.
-            **kwargs (dict): The kwargs.
-        """
+    def __init__(self):
+        """Initialize."""
+
+        LOGGER.debug('Initializing SecurityCenterClient')
         self.repository = SecurityCenterRepositoryClient()
 
     def create_finding(self, organization_id, finding):
         """Creates a finding in CSCC.
+
         Args:
-            organization_id (str): The id of the organization.
+            organization_id (str): The id prefixed with 'organizations/'.
+            finding (dict): Forseti violation in CSCC format.
         """
         try:
-            response = self.repository.findings.create(organization_id, finding)
-            LOGGER.debug('Created finding in CSCC: %s', list(response))
-            return
+            LOGGER.debug('Creating finding.')
+            response = self.repository.findings.create(
+                arguments={
+                    'body': {'sourceFinding': finding},
+                    'orgName': organization_id
+                }
+            )
+            LOGGER.debug('Created finding response in CSCC: %s', list(response))
         except (errors.HttpError, HttpLib2Error) as e:
-            LOGGER.info('>>>>> %s', e)
-            #if _is_status_not_found(e):
-            #    return []
-            #raise api_errors.ApiExecutionError(project_id, e)
-            raise api_errors.ApiExecutionError(e)
+            LOGGER.error(
+                'Unable to create resource:\n%s\n'
```

Comment:
Maybe 'Unable to create CSCC finding:' is easier to understand in this case?

Reply:
Done